### ISSUE TITLE
error on internal find command

### ISSUE THREAD

**Issue by socrat3** [Meliox/FTPauto#29]
when i give tho command, i have error on find command

bash ftpauto.sh --user=soc --path=/home/soc/tmp/ATTfriends.17.12.10.Elektra.Rose.2160p.MP4-KTR/

LOG
```
+ echo 'INFO: Information level: 1'
+ updateChecker
++ date +%s
++ echo 1512934845 + '14*24*60*60'
++ bc
+ [[ 1512938162 -gt 1514144445 ]]
+ [[ -n '' ]]
+ load_user
+ [[ -z soc ]]
+ [[ -n soc ]]
+ [[ -f /home/soc/FTPauto/users/soc/config ]]
+ username=soc
+ config_name=/home/soc/FTPauto/users/soc/default
+ loadDependency DConfig
+ local LoadPath=
+ case "$1" in
+ LoadPath=/home/soc/FTPauto/users/soc/config
+ source /home/soc/FTPauto/users/soc/config
++ config_version=6
++ transferetype=upftp
++ ftpincomplete=
++ ftpcomplete=/www.pino.com/prova/
++ ftpuser1=xxxx@xxx.it
++ ftppass1=ascccqw1234
++ ftphost1=ftp.pino.com
++ ftpport1=21
++ ftpssl1=true
++ ftpcustom1='set ssl:verify-certificate no'
++ ftpuser2=123
++ ftppass2=123
++ ftphost2=ftp.pino.it
++ ftpport2=21
++ ftpssl2=false
++ ftpcustom2=
++ logrotate=false
++ lognumber=50
++ send_option='(default)'
++ splitsize=1000
++ create_sfv=false
++ parallel=1
++ continue_queue=true
++ retries=1
++ retry_download=10
++ retry_download_max=1
++ force=true
++ confirm_transfer=false
++ confirm_online=false
++ exclude_array=
++ ftpsizemanagement=false
++ totalmb=14950
++ critical=100
++ sleeptime=2
++ exec_post=
++ allow_background=false
++ exec_pre=
++ sort=false
++ push_token=
++ push_user=
+ echo 'INFO: User: soc'
+ [[ 6 -ne 6 ]]
+ loadDependency DSetup
+ local LoadPath=
+ case "$1" in
+ LoadPath=/home/soc/FTPauto/dependencies/setup.sh
+ source /home/soc/FTPauto/dependencies/setup.sh
+ setup
+ PATH=/home/soc/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
++ which lftp
+ lftp=/usr/local/bin/lftp
++ which rar2fs
+ rar2fs=
+ queue_file=/home/soc/FTPauto/run/soc.queue
+ lockfile=/home/soc/FTPauto/run/soc.lck
+ logfile=/home/soc/FTPauto/users/soc/log
+ oldlogfile=/home/soc/FTPauto/users/soc/log.old
+ ftptransfere_file=/home/soc/FTPauto/run/soc.ftptransfere
+ ftptransfere_processbar=/home/soc/FTPauto/run/soc.ftpprocessbar
+ ftplogin_file1=/home/soc/FTPauto/run/soc.ftplogin1
+ ftplogin_file2=/home/soc/FTPauto/run/soc.ftplogin2
+ ftpfreespace_file=/home/soc/FTPauto/run/soc.ftpfreespace
+ ftpalive_file=/home/soc/FTPauto/run/soc.ftpalive
+ ftpcheck_testfile=/home/soc/FTPauto/run/soc.testfile
+ ftpcheck_file=/home/soc/FTPauto/run/soc.ftpcheck
+ proccess_bar_file=/home/soc/FTPauto/run/soc.transfered.info
+ ftp_size_file=/home/soc/FTPauto/run/soc.ftpsize.info
+ log_control=/home/soc/FTPauto/run/soc.controllog
+ ftpmaindebugfile=/home/soc/FTPauto/run/soc.ftpmain.debug
+ lftpdebug=/home/soc/FTPauto/run/soc.lftpdebug
+ lftptransfersize=/home/soc/FTPauto/run/soc.ftptransfersize
+ transfersize=/home/soc/FTPauto/run/soc.transfersize
+ transfersize2=/home/soc/FTPauto/run/soc.transfersize2
+ lftptransfersize2=/home/soc/FTPauto/run/soc.lftptransfersize2
+ ftplist_file=/home/soc/FTPauto/run/soc.lftplist
+ ftp_content=/home/soc/FTPauto/run/soc.ftpcontent
+ '[' '!' -e /home/soc/FTPauto/users/soc/log ']'
+ echo 'INFO: Option(s): download' start
+ main
+ safelock=true
+ case "${option[0]}" in
+ safelock=false
+ [[ -z '' ]]
+ source=CONSOLE
+ [[ start == \s\t\a\r\t ]]
+ start_ftpmain
+ loadDependency DFtpMain
+ local LoadPath=
+ case "$1" in
+ LoadPath=/home/soc/FTPauto/dependencies/ftp_main.sh
+ source /home/soc/FTPauto/dependencies/ftp_main.sh
+ [[ 1 -eq 1 ]]
+ start_main --user=soc --path=/home/soc/tmp/ATTfriends.17.12.10.Elektra.Rose.2160p.MP4-KTR
+ :
+ case "$1" in
+ user=soc
+ shift
+ :
+ case "$1" in
+ filepath=/home/soc/tmp/ATTfriends.17.12.10.Elektra.Rose.2160p.MP4-KTR
+ shift
+ :
+ case "$1" in
+ break
+ [[ -z /home/soc/tmp/ATTfriends.17.12.10.Elektra.Rose.2160p.MP4-KTR ]]
++ find /home/soc/tmp/ATTfriends.17.12.10.Elektra.Rose.2160p.MP4-KTR -type d
+ [[ -z /home/soc/tmp/ATTfriends.17.12.10.Elektra.Rose.2160p.MP4-KTR
/home/soc/tmp/ATTfriends.17.12.10.Elektra.Rose.2160p.MP4-KTR/Sample ]]
++ find /home/soc/tmp/ATTfriends.17.12.10.Elektra.Rose.2160p.MP4-KTR -type f -print
++ head -n 1
+ [[ -z /home/soc/tmp/ATTfriends.17.12.10.Elektra.Rose.2160p.MP4-KTR/atkgfs.17.12.10.elektra.rose.4k.r01 ]]
+ [[ '' == true ]]
+ [[ '' == \t\r\u\e ]]
+ lockfile
+ [[ -f /home/soc/FTPauto/run/soc.lck ]]
+ echo
+ echo
+ echo
+ echo
+ echo
+ sed '1c 14726' -i /home/soc/FTPauto/run/soc.lck
+ echo 'INFO: Process id: 14726'
+ echo 'INFO: Transfer-option: upftp'
+ loadDependency DSetup
+ local LoadPath=
+ case "$1" in
+ LoadPath=/home/soc/FTPauto/dependencies/setup.sh
+ source /home/soc/FTPauto/dependencies/setup.sh
+ loadConfig
+ loadDependency DConfig
+ local LoadPath=
+ case "$1" in
+ LoadPath=/home/soc/FTPauto/users/soc/config
+ source /home/soc/FTPauto/users/soc/config
++ config_version=6
++ transferetype=upftp
++ ftpincomplete=
++ ftpcomplete=/www.salvatorecrapanzano.com/prova/
++ ftpuser1=ccc
++ ftppass1=wdsdsd
++ ftphost1=sdsd
++ ftpport1=21
++ ftpssl1=true
++ ftpcustom1='set ssl:verify-certificate no'
++ ftpuser2=123
++ ftppass2=123
++ ftphost2=ftp.pino.it
++ ftpport2=21
++ ftpssl2=false
++ ftpcustom2=
++ logrotate=false
++ lognumber=50
++ send_option='(default)'
++ splitsize=1000
++ create_sfv=false
++ parallel=1
++ continue_queue=true
++ retries=1
++ retry_download=10
++ retry_download_max=1
++ force=true
++ confirm_transfer=false
++ confirm_online=false
++ exclude_array=
++ ftpsizemanagement=false
++ totalmb=14950
++ critical=100
++ sleeptime=2
++ exec_post=
++ allow_background=false
++ exec_pre=
++ sort=false
++ push_token=
++ push_user=
+ setup
+ PATH=/home/soc/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
++ which lftp
+ lftp=/usr/local/bin/lftp
++ which rar2fs
+ rar2fs=
+ queue_file=/home/soc/FTPauto/run/soc.queue
+ lockfile=/home/soc/FTPauto/run/soc.lck
+ logfile=/home/soc/FTPauto/users/soc/log
+ oldlogfile=/home/soc/FTPauto/users/soc/log.old
+ ftptransfere_file=/home/soc/FTPauto/run/soc.ftptransfere
+ ftptransfere_processbar=/home/soc/FTPauto/run/soc.ftpprocessbar
+ ftplogin_file1=/home/soc/FTPauto/run/soc.ftplogin1
+ ftplogin_file2=/home/soc/FTPauto/run/soc.ftplogin2
+ ftpfreespace_file=/home/soc/FTPauto/run/soc.ftpfreespace
+ ftpalive_file=/home/soc/FTPauto/run/soc.ftpalive
+ ftpcheck_testfile=/home/soc/FTPauto/run/soc.testfile
+ ftpcheck_file=/home/soc/FTPauto/run/soc.ftpcheck
+ proccess_bar_file=/home/soc/FTPauto/run/soc.transfered.info
+ ftp_size_file=/home/soc/FTPauto/run/soc.ftpsize.info
+ log_control=/home/soc/FTPauto/run/soc.controllog
+ ftpmaindebugfile=/home/soc/FTPauto/run/soc.ftpmain.debug
+ lftpdebug=/home/soc/FTPauto/run/soc.lftpdebug
+ lftptransfersize=/home/soc/FTPauto/run/soc.ftptransfersize
+ transfersize=/home/soc/FTPauto/run/soc.transfersize
+ transfersize2=/home/soc/FTPauto/run/soc.transfersize2
+ lftptransfersize2=/home/soc/FTPauto/run/soc.lftptransfersize2
+ ftplist_file=/home/soc/FTPauto/run/soc.lftplist
+ ftp_content=/home/soc/FTPauto/run/soc.ftpcontent
++ echo /home/soc/tmp/ATTfriends.17.12.10.Elektra.Rose.2160p.MP4-KTR
++ sed 's/\\./ /g'
+ filepath=/home/soc/tmp/ATTfriends.17.12.10.Elektra.Rose.2160p.MP4-KTR
+ main /home/soc/tmp/ATTfriends.17.12.10.Elektra.Rose.2160p.MP4-KTR
+ filepath=/home/soc/tmp/ATTfriends.17.12.10.Elektra.Rose.2160p.MP4-KTR
+ transfer_path=/home/soc/tmp/ATTfriends.17.12.10.Elektra.Rose.2160p.MP4-KTR
++ basename /home/soc/tmp/ATTfriends.17.12.10.Elektra.Rose.2160p.MP4-KTR
+ orig_name=ATTfriends.17.12.10.Elektra.Rose.2160p.MP4-KTR
+ [[ -f /home/soc/tmp/ATTfriends.17.12.10.Elektra.Rose.2160p.MP4-KTR ]]
+ [[ -d /home/soc/tmp/ATTfriends.17.12.10.Elektra.Rose.2160p.MP4-KTR ]]
+ tempdir=/home/soc/FTPauto/run/soc-temp/ATTfriends.17.12.10.Elektra.Rose.2160p.MP4-KTR-temp/
++ date +%s
+ ScriptStartTime=1512938162
++ date --date=@1512938162 +%d/%m/%y-%a-%H:%M:%S
+ echo 'INFO: Process start-time: 10/12/17-dom-21:36:02'
+ echo 'INFO: Preparing transfer: /home/soc/tmp/ATTfriends.17.12.10.Elektra.Rose.2160p.MP4-KTR'
+ echo 'INFO: Lunched from: CONSOLE'
+ queue add
+ local option i old_id
+ option=
+ case "$1" in
+ [[ '' == true ]]
+ id_old=
+ [[ -e /home/soc/FTPauto/run/soc.queue ]]
+ id=1
+ get_size /home/soc/tmp/ATTfriends.17.12.10.Elektra.Rose.2160p.MP4-KTR
+ [[ -e /home/soc/FTPauto/run/soc.queue ]]
+ [[ '' == failed ]]
+ [[ '' == end ]]
+ echo 'INFO: Queueid: 1'
++ date +%d/%m/%y-%a-%H:%M:%S
+ echo '1|CONSOLE|/home/soc/tmp/ATTfriends.17.12.10.Elektra.Rose.2160p.MP4-KTR||0MB|false|10/12/17-dom-21:36:02'
+ echo 'INFO: Simultaneous transfers: 1'
+ get_size /home/soc/tmp/ATTfriends.17.12.10.Elektra.Rose.2160p.MP4-KTR
+ local dir n count i exp path t_size
+ dir=/home/soc/tmp/ATTfriends.17.12.10.Elektra.Rose.2160p.MP4-KTR
+ [[ upftp == downftp ]]
+ [[ upftp == fxp ]]
+ [[ upftp == upftp ]]
++ du -bsL /home/soc/tmp/ATTfriends.17.12.10.Elektra.Rose.2160p.MP4-KTR
++ awk '{print $1}'
+ directorysize=1900088755
++ echo 'scale=2; 1900088755 / (1024*1024)'
++ bc
+ size=1812.06
+ echo 'INFO: Size to transfere: 1812.06MB'
+ [[ 1 -gt 0 ]]
+ exclude_expression=()
+ n=1
+ for i in "${exclude_array[@]}"
+ exclude_expression+=("-iname *$i*")
+ [[ 1 -lt 1 ]]
+ let n++
+ exclude_expression='-iname **'
++ awk -F : '{sum+=$NF} END { printf ("%0.0f\n", sum)}'
++ find -L /home/soc/tmp/ATTfriends.17.12.10.Elektra.Rose.2160p.MP4-KTR '!' '(' -iname dependencies ftpauto.sh install.sh LICENCE logga.txt log.txt plugins README.md reg.log run users ')' -type f -printf '%s\n'
find: i percorsi devono precedere le espressioni: ftpauto.sh
Try 'find --help' for more information.
+ directorysize=0
++ echo 'scale=2; 0 / (1024*1024)'
++ bc
+ size=0
+ echo 'INFO: Updated size to transfere(regex used): 0MB'
+ [[ -n '' ]]
+ loadDependency DFtpLogin
+ local LoadPath=
+ case "$1" in
+ LoadPath=/home/soc/FTPauto/dependencies/ftp_login.sh
+ source /home/soc/FTPauto/dependencies/ftp_login.sh
+ ftp_login 1
+ local number OIFS IFS ftpcustom ftpssl ftpuser ftppass ftphost ftppass ftploginfile
+ number=1
+ ftpcustom=ftpcustom1
+ ftpssl=ftpssl1
+ ftpuser=ftpuser1
+ ftppass=ftppass1
+ ftphost=ftphost1
+ ftpport=ftpport1
+ ftploginfile=ftplogin_file1
+ echo 'set net:timeout 10'
+ echo 'set net:max-retries 3'
+ echo 'set net:reconnect-interval-base 10'
+ echo 'set net:reconnect-interval-multiplier 1'
+ echo 'set net:reconnect-interval-max 60'
+ (( verbose ))
+ echo 'debug 8 -t -o /home/soc/FTPauto/run/soc.lftpdebug'
+ [[ true == true ]]
+ echo 'set ftp:ssl-force true'
+ echo 'set ssl:verify-certificate false'
+ [[ -n set ssl:verify-certificate no ]]
+ OIFS=
+ IFS=';'
+ for i in "${!ftpcustom[@]}"
+ echo 0
+ IFS=
+ [[ upftp =~ upftp ]]
+ echo 'open -u ddsds@sdfdf sdfdsfdfcom -p 21'
+ [[ false == \t\r\u\e ]]
+ [[ false == \t\r\u\e ]]
+ echo 'INFO: Sendoption: (default)'
+ [[ (default) == \s\p\l\i\t ]]
+ [[ (default) == \v\i\d\e\o ]]
+ [[ false == \t\r\u\e ]]
+ [[ -n '' ]]
+ delay
+ local current_epoch target_epoch sleep_seconds timediff
+ [[ -n '' ]]
+ [[ -n '' ]]
+ ftp_transfere
+ local lftp_exclude quittime waittime
+ [[ 1 -gt 0 ]]
+ [[ '' = directory ]]
+ [[ -d /home/soc/tmp/ATTfriends.17.12.10.Elektra.Rose.2160p.MP4-KTR ]]
+ (( i=0 ))
+ (( i<1 ))
+ [[ 0 -gt 0 ]]
+ lftp_exclude='^.**'
+ (( i++ ))
+ (( i<1 ))
+ lftp_exclude='^.**$'
+ echo 'set mirror:exclude-regex "^.**$"'
+ echo 'set mirror:no-empty-dirs true'
+ [[ upftp == \d\o\w\n\f\t\p ]]
+ [[ upftp == \u\p\f\t\p ]]
+ cat /home/soc/FTPauto/run/soc.ftplogin1
+ echo 'mkdir -p "/www.salvatorecrapanzano.com/prova/"'
+ [[ -n '' ]]
+ echo 'set cmd:fail-exit true'
+ [[ -f /home/soc/tmp/ATTfriends.17.12.10.Elektra.Rose.2160p.MP4-KTR ]]
+ [[ -d /home/soc/tmp/ATTfriends.17.12.10.Elektra.Rose.2160p.MP4-KTR ]]
+ [[ -n '' ]]
+ [[ -z '' ]]
+ echo 'queue mirror --no-umask -p --parallel=1 -c -RL "/home/soc/tmp/ATTfriends.17.12.10.Elektra.Rose.2160p.MP4-KTR" "/www.salvatorecrapanzano.com/prova/"'
+ echo wait
+ [[ -n '' ]]
+ echo quit
+ [[ '' != \t\r\u\e ]]
+ ftp_transfer_process start
+ local pid_f_process
+ case "$1" in
++ date +%s
+ TransferStartTime=1512938162
+ pid_f_process=14777
+ ftp_processbar
+ sed '3c 14777' -i /home/soc/FTPauto/run/soc.lck
+ local transfered_size ProgressTimeNew TransferredNew TransferredNewMB TotalTimeDiff TimeDiff percentage speed eta etatime SpeedOld sum SpeedAverage cols percentagebarlength string string2 TransferredOld ProgressTimeOld
+ [[ '' != \t\r\u\e ]]
+ sleep 5
++ date --date=@1512938162 +%d/%m/%y-%a-%H:%M:%S
+ echo -e '\e[00;37mINFO: \e[00;32mTransfer started: 10/12/17-dom-21:36:02\n\e[00m'
+ pid_transfer=14781
+ sed '2c 14781' -i /home/soc/FTPauto/run/soc.lck
+ /usr/local/bin/lftp -f /home/soc/FTPauto/run/soc.ftptransfere
+ wait 14781
+ pid_transfer_status=0
++ date +%s
+ TransferEndTime=1512938164
+ [[ 0 -ne 0 ]]
+ ftp_transfer_process stop-process-bar
+ local pid_f_process
+ case "$1" in
++ sed -n 3p /home/soc/FTPauto/run/soc.lck
+ kill 14777
/home/soc/FTPauto/dependencies/ftp_main.sh: riga 180: 14777 Terminato               ftp_processbar
++ sed -n 3p /home/soc/FTPauto/run/soc.lck
+ wait 14777
++ sed -n 4p /home/soc/FTPauto/run/soc.lck
+ kill
++ sed -n 4p /home/soc/FTPauto/run/soc.lck
+ wait
++ date --date=@1512938164 +%d/%m/%y-%a-%H:%M:%S
+ echo -e '\n\e[00;37mINFO: \e[00;32mTransfer ended: 10/12/17-dom-21:36:04\e[00m'
+ [[ false == \t\r\u\e ]]
+ [[ '' != true ]]
+ logrotate
+ [[ '' != \t\r\u\e ]]
+ transferTime=2
++ printf %02dh:%02dm:%02ds 0 0 2
+ transferTime2=00h:00m:02s
++ sed -n 5p /home/soc/FTPauto/run/soc.lck
+ SpeedAverage=
++ date --date=@1512938162 +%d/%m/%y-%a-%H:%M:%S
+ sed '7i 10/12/17-dom-21:36:02|CONSOLE|ATTfriends.17.12.10.Elektra.Rose.2160p.MP4-KTR|0MB|00h:00m:02s|MB/s' -i /home/soc/FTPauto/users/soc/log
+ lognumber=57
+ [[ false == \t\r\u\e ]]
+ '[' 57 -ne 0 ']'
+ sed '57,$d' -i /home/soc/FTPauto/users/soc/log
++ awk 'BEGIN{FS="|";OFS=" "}NR==2{print $1}' /home/soc/FTPauto/users/soc/log
++ cut '-d ' -f4
+ totaldl=
+ totaldl=
+ [[ -z '' ]]
+ totaldl=0
++ echo '0 + 0'
++ bc
+ totaldl=0
++ awk 'BEGIN{FS="|";OFS=" "}NR==2{print $1}' /home/soc/FTPauto/users/soc/log
++ cut '-d ' -f6
+ totalrls=
++ echo ' + 1'
++ bc
(standard_in) 1: syntax error
+ totalrls=
++ awk 'BEGIN{FS="|";OFS=" "}NR==2{print $1}' /home/soc/FTPauto/users/soc/log
++ cut '-d ' -f10
+ totaldltime=
++ awk 'BEGIN{split("",a,":"); print a[1]*(60*60*24)+a[2]*(60*60)+a[3]*60+a[4];}'
+ totaldltime_seconds=0
++ echo '0 + 2'
++ bc
+ totaldltime=2
++ printf %02dd:%02dh:%02dm:%02ds 0 0 0 2
+ totaldltime=00d:00h:00m:02s
+ sed '1s#.*#*****  FTPauto 0.5.4#' -i /home/soc/FTPauto/users/soc/log
+ sed '2s#.*#*****  STATS: 0MB in  transfers in 00d:00h:00m:02s#' -i /home/soc/FTPauto/users/soc/log
+ sed '3s#.*#*****  FTP INFO: N/A#' -i /home/soc/FTPauto/users/soc/log
++ date
+ sed '4s#.*#*****  LASTDL: dom 10 dic 2017, 21.36.05, CET|ATTfriends.17.12.10.Elektra.Rose.2160p.MP4-KTR|MB/s#' -i /home/soc/FTPauto/users/soc/log
+ sed '5s#.*#*****  #' -i /home/soc/FTPauto/users/soc/log
+ cleanup session
+ local array
+ case "$1" in
+ [[ '' == \t\r\u\e ]]
+ [[ '' == \t\r\u\e ]]
+ array=("$ftplogin_file1" "$ftplogin_file2" "$ftptransfere_file" "$ftp_size_file" "$ftpfreespace_file" "$lftptransfersize" "$lftptransfersize2" "$transfersize" "$transfersize2" "$proccess_bar_file" "$ftpalive_file" "$ftpcheck_file" "$ftpcheck_testfile" "$ftptransfere_processbar")
+ removeClean /home/soc/FTPauto/run/soc.ftplogin1 /home/soc/FTPauto/run/soc.ftplogin2 /home/soc/FTPauto/run/soc.ftptransfere /home/soc/FTPauto/run/soc.ftpsize.info /home/soc/FTPauto/run/soc.ftpfreespace /home/soc/FTPauto/run/soc.ftptransfersize /home/soc/FTPauto/run/soc.lftptransfersize2 /home/soc/FTPauto/run/soc.transfersize /home/soc/FTPauto/run/soc.transfersize2 /home/soc/FTPauto/run/soc.transfered.info /home/soc/FTPauto/run/soc.ftpalive /home/soc/FTPauto/run/soc.ftpcheck /home/soc/FTPauto/run/soc.testfile /home/soc/FTPauto/run/soc.ftpprocessbar
+ local array
+ array=("$@")
+ for i in "${array[@]}"
+ rm -f /home/soc/FTPauto/run/soc.ftplogin1
+ for i in "${array[@]}"
+ rm -f /home/soc/FTPauto/run/soc.ftplogin2
+ for i in "${array[@]}"
+ rm -f /home/soc/FTPauto/run/soc.ftptransfere
+ for i in "${array[@]}"
+ rm -f /home/soc/FTPauto/run/soc.ftpsize.info
+ for i in "${array[@]}"
+ rm -f /home/soc/FTPauto/run/soc.ftpfreespace
+ for i in "${array[@]}"
+ rm -f /home/soc/FTPauto/run/soc.ftptransfersize
+ for i in "${array[@]}"
+ rm -f /home/soc/FTPauto/run/soc.lftptransfersize2
+ for i in "${array[@]}"
+ rm -f /home/soc/FTPauto/run/soc.transfersize
+ for i in "${array[@]}"
+ rm -f /home/soc/FTPauto/run/soc.transfersize2
+ for i in "${array[@]}"
+ rm -f /home/soc/FTPauto/run/soc.transfered.info
+ for i in "${array[@]}"
+ rm -f /home/soc/FTPauto/run/soc.ftpalive
+ for i in "${array[@]}"
+ rm -f /home/soc/FTPauto/run/soc.ftpcheck
+ for i in "${array[@]}"
+ rm -f /home/soc/FTPauto/run/soc.testfile
+ for i in "${array[@]}"
+ rm -f /home/soc/FTPauto/run/soc.ftpprocessbar
+ [[ -d /home/soc/FTPauto/run/soc-temp ]]
+ echo -e 'INFO: Cleanup done'
+ [[ -n '' ]]
+ [[ -n '' ]]
++ date +%s
+ ScriptEndTime=1512938165
+ TotalTransferTime=3
+ [[ '' == true ]]
+ echo -e '\e[00;37mINFO: \e[00;32mTransfere finished\e[00m'
+ echo '                       Name: ATTfriends.17.12.10.Elektra.Rose.2160p.MP4-KTR'
+ echo '                       Size: 0 MB'
+ echo '                      Speed:  MB/s'
+ echo '              Transfer time: 00h:00m:02s'
++ date --date=@1512938162 +%d/%m/%y-%a-%H:%M:%S
+ echo '                 Start time: 10/12/17-dom-21:36:02'
++ date --date=@1512938165 +%d/%m/%y-%a-%H:%M:%S
+ echo '                   End time: 10/12/17-dom-21:36:05'
++ printf %02dh:%02dm:%02ds 0 0 3
+ echo '                 Total time: 00h:00m:03s'
+ [[ '' != true ]]
+ queue remove
+ local option i old_id
+ option=
+ case "$1" in
+ sed '/^1/d' -i /home/soc/FTPauto/run/soc.queue
+ [[ true == true ]]
+ queue next
+ local option i old_id
+ option=
+ case "$1" in
+ [[ -f /home/soc/FTPauto/run/soc.queue ]]
++ cat /home/soc/FTPauto/run/soc.queue
+ [[ -n '' ]]
+ echo '----------------------- Empty queue -----------------------'
+ [[ -f /home/soc/FTPauto/run/soc.queue ]]
+ rm /home/soc/FTPauto/run/soc.queue
+ echo -e 'INFO: Queue is empty. Program will end\n'
+ cleanup end
+ local array
+ case "$1" in
+ removeClean /home/soc/FTPauto/run/soc.lck
+ local array
+ array=("$@")
+ for i in "${array[@]}"
+ rm -f /home/soc/FTPauto/run/soc.lck
+ sed '5s#.*#***************************	#' -i /home/soc/FTPauto/users/soc/log
+ echo -e '\e[00;32mExiting successfully...\e[00m\n'
+ exit 0
```

**Comment by Meliox** [comment]
Hmm. At least the script finishes correctly.
Could you attach your config file because what I see is that the array "exclude_array=" is empty, but it is actually evaluated as 1:
```
+ echo 'INFO: Size to transfere: 1812.06MB'
+ [[ 1 -gt 0 ]]
```
Remember to remove confidential information.

**Comment by socrat3** [comment]
#This the the configuration file for FTPauto
config_version="6"
#Place the file in /home/soc/FTPauto/run/'soc'/config and load with --user='soc'

#HOWTO: Edit the info between the qoutes "TEST", here the word TEST

#### FTP server Setup ####
 # If you just want the server to send <ITEM> to your ftp, edit the options below
transferetype="upftp" # Determine how to transfer file: Either send or receive from ftp or fxp them to another server

 # These directories are where you want to download/send the item. REMEMBER TRAILING SLASH
ftpincomplete="" # incomplete directory. Leave empty if no incomplete directory should be used
ftpcomplete="/www.sssss.com/prova/" # complete directory.

#### DOWN/UP MODE ####
 # If you just want to send/receive items, change these
ftpuser1="sssssdd@sssss.it" # username
ftppass1="ddd" # password
ftphost1="ftp.sssss.com" # ip address for ftp server
ftpport1="21" # ftp port
ftpssl1="true" # Set to true to use ftps else set it to false
ftpcustom1="set ssl:verify-certificate no" # Enter settings separated by ';', eg. set cache:expire 1;set cache:size 2

#### FXP MODE ####
 # If you just want to send/receive items from one server to another, change these also! In FXP mode, ftphost1 is source and ftphost2 receiver
ftpuser2="123" # username
ftppass2="123" # password
ftphost2="ftp.pino.it" # ip address for ftp server
ftpport2="21" # ftp port
ftpssl2="false" # use ftps or not
ftpcustom2="" # Enter settings separated by ';', eg. set cache:expire 1; set cache:size 2

#### Log settings ###
logrotate="false" #enabled logrotating to move old to log.old
lognumber="50" #how many transfers to save in log before moving to log.old. 0 for disabled

#### Transfer settings ####

### Filehandling
 # Splitting files if filesize exceed MB. Some FTP servers disconnect after a certain amount of time is there is no
 # activity. These settings only work if the server handling the script also sends the files, i.e. in upftp and upfxp mode!
send_option="default" # Can be configured to send only videofile, split files according to settings or simply transfer the , default. If videofile or sizelimit are not met, then the files will be transfered as default - without any modifications.
splitsize="1000" # How large the rarparts should be in MB
create_sfv="false" # Create sfv for rarfiles

### Transfer settings

## General settings
parallel="1" # how many simultaneous transfers to download with
continue_queue="true" # Script will continue downloading if something is queued
retries="1" # How many times should the transfer be tried, before giving up
retry_download="10" # retry again in minutes after minimum space is reached OR server is offline.
retry_download_max="1" # retry for how many minites, before quitting. Recommended 30 mins. For each try 3 tries to establish connection with furthermore be tried

## Extra settings
force="true" # Transfer regardless of lockfiles/other transfers
confirm_transfer="false" # Try to confirm transfer
confirm_online="false" # Try to confirm that server is online/writeable before doing anything
exclude_array="" # Ignore certain files with name matching, format is ( "word1" "word2" )

## Extra settings
 # To enable FTP space info, ftpsizemanagement has to be set to true
ftpsizemanagement="false" # will confirm enough free space in dir according to settings
totalmb="14950" # total ftp space in mb
critical="100" # minimum space before aborting transfer in mb

## Processbar settings
 # Processbar shows how the transfer is proceeding, gives eta. etc.
sleeptime="2"  # how often to provide progress information. Time in seconds

## Miscellaneous settings
exec_post="" #Execute external command upon finish. See --help exec_pre for more info
allow_background="false" # don't wait for exec to finish. ONLY for exec_post
exec_pre="" #Execute external command before starting. See --help for more info
sort="false" # Sort files into DVD/TV/etc/ or like defined in --sort=DIRECTORY. Changes can be made in the file /dependencies/sorting.sh

#### Push notifications ####
 # Create a user at https://pushover.net/ and enter details below
 # Leave push_user empty if you don't use it
push_token=""
push_user=""

**Comment by Meliox** [comment]
Ok, thanks. You can fix it by writing exclude_array=() instead in your config.

**Comment by Meliox** [comment]
Did it work for you?

**Comment by Meliox** [comment]
Fixed in Version 0.5.5
